docs/source/asking_questions.rst

```diff
@@ -62,9 +65,12 @@ input values and/or an input manifest.
 
 You can also set the following options when you call :mod:`Child.ask <octue.resources.child.Child.ask>`:
 
+- ``children`` - if provided, these children will override any default children the child asks questions to as part of its analysis (see :ref:`this subsection below <overriding_children>`).
```

Comment:
If the child has children of its own (i.e. grandchildren of this service), this optional section can be used to override the child's "default" children. This allows you to specify, say, particular versions of grandchildren to use.